app/views/document_lead_image/_image_list.html.erb

```diff
@@ -0,0 +1,38 @@
+<table>
```

Comment:
Perhaps we could give @alex-ju a hand here and put this in a component, so at least the boilerplate is there.